typings.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module 'react-media' {
+  interface MediaProps {
+    query: string;
```

Comment:
`query` should be a string or an object with string keys and values of number, string or boolean. looks like react-media will pass a string query to `window.matchMedia(query)`, and will pass an object query directly to json2mq:
https://github.com/akiran/json2mq

maybe this:
```
export interface MediaQueryObject {
    [id: string]: boolean | number | string;
}

export interface MediaProps {
    query: string | MediaQueryObject;
    ...
}
```

Reply:
looks like `query` can also accept an array of objects, so maybe this is better:

```
export interface MediaQueryObject {
    [id: string]: boolean | number | string;
}

export interface MediaProps {
    query: string | MediaQueryObject | MediaQueryObject[];
    ...
}
```